build.gradle.kts

```diff
@@ -3,54 +3,50 @@ plugins {
     id("org.jetbrains.changelog") version "1.3.1"
 }
 
+version = "0.1.1"
+group = "com.github.zeckie"
+
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    implementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
-    implementation("org.junit.jupiter:junit-jupiter-engine:5.8.1")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.2")
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.8.2")
 }
 
-tasks.test {
-    useJUnitPlatform()
-}
-
-version = "0.1.1"
-group = "com.github.zeckie"
-
-// Work out current java version
-val MIN_JAVA_VER = 9
-val fullVersion = System.getProperty("java.version")
-val majorVersion = Integer.parseInt(fullVersion.substring(0, fullVersion.indexOf('.')))
-System.out.println("Java major version: " + majorVersion)
+val minimumJavaVersion = JavaVersion.VERSION_1_9
 
-if (majorVersion < MIN_JAVA_VER) {
-    java {
-        toolchain {
-            languageVersion.set(JavaLanguageVersion.of(MIN_JAVA_VER))
-        }
-    }
+if (JavaVersion.current() < minimumJavaVersion) {
+    java.toolchain.languageVersion.set(JavaLanguageVersion.of(minimumJavaVersion.majorVersion))
 } else {
-    if (majorVersion > 9) {
+    if (JavaVersion.current() > JavaVersion.VERSION_1_9) {
```

Comment:
I don't think there was ever a java version 1.9 (due to JEP223)

Reply:
From the implementation:
```java
    // Since Java 9, version should be X instead of 1.X
    // However, to keep backward compatibility, we change from 11
```